index.js

```diff
@@ -26,10 +26,16 @@ module.exports = (function () {
    * @return {Promise} returns String passage
    */
   bible.get = function (psg, ver) {
-    if (typeof psg !== 'string') { throw new TypeError('Expected a string'); }
+    if (typeof psg !== 'string') {
+      throw new TypeError('Passage should be a string');
+    }
+    if (typeof ver !== 'undefined' && typeof ver !== 'string') {
+      throw new TypeError('Version should be a string');
+    }
 
     var self = this;
     var v = ver || 'asv';
```

Comment:
Ahh, I see the confusion now! 

The check above says if `ver` _is defined_ then it must be of type string, but it still allows `ver` to be passed as undefined and therefore use the default of `'asv'`.

If you'd prefer to simply require that `ver` be defined always then that is a valid option, but I'd suggest against it as it is most definitely a breaking change and could significantly impact usages of this repo in the wild. Plus I almost see making `ver` a require option as a regression as it make the entire library less flexible.

Sorry for being so unclear XD